src/main/java/mate/jdbc/services/impl/ManufactureServiceImpl.java

```diff
@@ -0,0 +1,45 @@
+package mate.jdbc.services.impl;
+
+import java.util.List;
+import java.util.Optional;
+import mate.jdbc.dao.ManufacturerDao;
+import mate.jdbc.dao.impl.ManufacturerDaoImpl;
+import mate.jdbc.lib.Inject;
+import mate.jdbc.lib.Service;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.services.ManufacturerService;
+
+@Service
+public class ManufactureServiceImpl implements ManufacturerService {
+    @Inject
+    private static final ManufacturerDao manufacturerDao = new ManufacturerDaoImpl();
```

Comment:
Remove the manual instantiation of ManufacturerDaoImpl, as the @Inject annotation is used to handle the dependency injection.